tests/benchmarks/power_consumption/main.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/gpio.h>
+
+#define LED1_NODE DT_ALIAS(led1)
+const struct gpio_dt_spec led = GPIO_DT_SPEC_GET(LED1_NODE, gpios);
+static bool state = true;
```

Comment:
Why not inside thread?

Reply:
there is no specific reason for it, can move it to thread definition.

Reply:
Never mind, my wrong assumtion.